### ISSUE TITLE
MPE 0.14.0 generate the front matter into pdf

### ISSUE THREAD

**Issue by gepcel** [shd101wyy/markdown-preview-enhanced#496]
Updated to 0.14.0, found an issue that now MPE put the front matter into the pdf file (generated by pandoc). This was not the behavior of the last version.

My `.md` file was like:

```markdown
---
output:
    pdf_document:
        latex_engine: xelatex
        number_sections: true
        toc: true
geometry: margin=1in
mainfont: 'Times New Roman'
fontsize: 12pt
---

# Chapter 1

## C1.1

C1.1 here

## C1.2

# Chapter 2

C2 here

```

And then I use MPE to get a pdf file (preview panel -> right click menu -> pandoc)，and the pdf was like (notice the front matter information as the read ink indicates):

![image](https://user-images.githubusercontent.com/4179106/28235058-80452e94-6939-11e7-818b-a13d038c0e36.png)

[test.md.txt](https://github.com/shd101wyy/markdown-preview-enhanced/files/1149938/test.md.txt)
[test.pdf](https://github.com/shd101wyy/markdown-preview-enhanced/files/1149937/test.pdf)

**Comment by shd101wyy** [comment]
Yes I found this issue. I will fix it tonight.

**Comment by shd101wyy** [comment]
Should be fixed now :)

**Comment by gepcel** [comment]
I'm not sure what is wrong, but I still got this with MPE 0.14.1.

**Comment by shd101wyy** [comment]
Did you restart atom after updating this package?

**Comment by gepcel** [comment]
Yes, I restart atom after updating.      
I'm using Windows 10 (v1703 to be specific)     
Yes, the preview display front-matter correctly.     
And atom v1.18.0 x64, mpe v0.14.1, pandoc v1.19.2.1.
![image](https://user-images.githubusercontent.com/4179106/28243531-699984fc-6a00-11e7-98e4-8b1871d26646.png)

And the pdf output still be: 

![image](https://user-images.githubusercontent.com/4179106/28243533-8b3a799a-6a00-11e7-8607-307c69cef494.png)

**Comment by shd101wyy** [comment]
@gepcel Should be fixed now. Cheers :)

**Comment by gepcel** [comment]
I can confirm it's fixed in 0.14.2
